includes/misc-functions.php

```diff
@@ -1971,16 +1978,23 @@ function give_goal_progress_stats( $form ) {
 			$actual = apply_filters( 'give_goal_donors_target_output', give_get_form_donor_count( $form->ID ), $form->ID, $form );
 			break;
 		default:
-			/**
-			 * Filter the form income.
-			 *
-			 * @since 1.8.8
-			 */
-			$actual = apply_filters( 'give_goal_amount_raised_output', (new DonationQuery())->form($form->ID)->sumIntendedAmount(), $form->ID, $form );
+
+            if ($usePlaceholder) {
+                $actual = $placeholder;
```

Comment:
This part confuses me a bit because we're assigning an html element to a variable that is supposed to be a numeric value right?  

See comment below, this might not be necessary

Reply:
I did simplify it, but we still need to prevent the `give_goal_amount_raised_output` from being triggered when using the placeholder.

Resolved in 7d8059274d5a01e5df31058fd2a1fec04ad245d2